ChatRPG/Services/GameStateManager.cs

```diff
@@ -70,14 +69,21 @@ private void UpdateStateFromResponse(Campaign campaign, OpenAiGptMessage message
                     _logger.LogInformation("Created event: \"{Name}\"", @event.Name);
                 }
             }
+
+            if (response.IsInCombat != null)
+            {
+                // TODO: this is not workings
+                // campaign.CombatMode = response.IsInCombat.GetValueOrDefault(false);
+                _logger.LogInformation("Combatmode: {CombatMode}", campaign.CombatMode);
+            }
```

Comment:
Jeg tænker det her er ret vigtigt at få styr på, da man gerne skal kunne skifte til en "Do" prompt og udelukkende "Rulle om bag et bord i dækning" i combat uden at tage en direkte attack action.

Reply:
der er styr på det yo

Reply:
it works now